scripts/provisioning-tests

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 set -e
+if ./scripts/only-ui-bumps.sh; then
```

Comment:
I'm wondering if it may be a good idea to check `RELEASE_TYPE` and actually run these and the rke provisioning tests for `final-rc`. I also agree that we shouldn't be relying on rcs to catch bugs, but it may be a good idea to run them for the final rc as a final sanity check before releasing.

Reply:
We currently don't have a distinction for rc or final-rc when we trigger it (its just another rc tag). There is no pattern that we can match to run the tests. Also the tests are disabled as it seems now due to continuous failures. But I do agree that we can run these for final-rc, just need to figure out how we are going to tell Drone its a final rc.

The goal of this change is to not spend unneeded resources (Drone runners + engineers time) on when UI is bumped. I would like to extend this filter to docs/README changes etc because that also does not require a build.

Reply:
Cool, mostly just wanted to get the wheels turning regarding it. I'm fine with merging in the meantime while we think of ways to run tests (even if it's manually updating an internal variable somewhere).